ios/FluentUI/Vnext/PersonaButton/PersonaButton.swift

```diff
@@ -0,0 +1,273 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// `MSFPersonaButtonState` contains PersonaButton properties in addition to a subset of the MSFAvatarState protocol.
+///
+/// `onTapAction` provides tap gesture for PersonaButton.
+///
+@objc public protocol MSFPersonaButtonState: MSFAvatarState {
```

Comment:
I believe the intent of implementing the properties below is to not inherit the MSFAvatarState?

Reply:
Ah what a mess this branch has become. Sorry, that's correct.